sdk/ml/azure-ai-ml/tests/internal/_utils.py

```diff
@@ -186,6 +190,33 @@
     PARAMETERS_TO_TEST,
 )))
 
+TEST_CASE_NAME_ENUMERATE = [TEST_CASE_NAME_ENUMERATE[2]]
```

Comment:
What's this logic for?

Reply:
Good catch! it's a test code and need to be removed